### DISCUSSION TITLE
[Request] User Topup Payment support 充值支付

### DISCUSSION THREAD

**Discussion by shawnmuggle** [lobehub/lobehub#4261] [General Question] [answered]
### 🥰 需求描述

What about adding a payment service on the user Account session ? 
Either subscription fee or pay as you go.

### 🧐 解决方案

1. Since it's a global userbase, we start with crypto payments topup
2. Users can topup their account by USDT and USDC payments (stablecoins)
3. Ideally, the money will be sent to the developer (self managed). Developer can pay OpenRouter for api key.

[a comment by an automated account is not shown]

**Comment by Sun-drenched** [comment, marked as answer]
Recharge payment and other commercial/profit functions should be available through the commercial version of Lobechat. You can contact the developer through the official community on Discord for details.
The community version of lobechat open source on GitHub should not include these functions

**Comment by arvinxx** [comment]
@Sun-drenched is right. The payment and user management admin is not suitable for the open source edition.

**Comment by Richard0620** [comment]
Hey folks — jumping in from OKX. We're working on [OnchainOS](https://github.com/okx/onchainos-skills
), an agent-native wallet SDK that supports USDT/USDC (and 20+ chains) out of the box.
Happy to open a PR that adds this as a [payment provider / plugin / tool] for you — no cost to maintainers, you review the code. If there's interest, we can post a short design proposal in the next day or two.
